src/logic/tictactoe.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+module.exports = class TicTacToe {
+  constructor () {
+    this.emptyToken = '-'
+    this.tieToken = 'T'
+    this.playerOne = 'X'
+    this.playerTwo = 'O'
+
+    // Start with a clean board
+    this.newGame()
+  }
+
+  newGame () {
+    // Reset the board
+    this.board = {
+      rows: [],
+      moves: 0,
+      maxMoves: 9,
+      currentPlayer: 'X'
+    }
+
+    // Fill the board with emptyTokens
```

Comment:
Would be simpler to initialize the board all at once i.e. something like
`board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']]` 
or
`board = [[emptyToken, emptyToken, emptyToken], [emptyToken, emptyToken, emptyToken], [emptyToken, emptyToken, emptyToken]]`

?